src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -36,6 +38,7 @@
   static final byte KEY_PATH_MAX_DEPTH = 10;
   static final byte PAIRING_MAX_CLIENT_COUNT = 1;
   static final byte UID_LENGTH = 16;
+  static final byte SAVED_DATA_SIZE = 155; // 5x compressed public key
```

Comment:
Are we sure 155 bytes is enough? I know for one, we should include 2 bytes for TLV headers before each pubKey. Does a musig script with 5 keys require any other data? I feel like you would need some script logic to determine M of N, or at minimum some metadata about what type of script data is stored.

If its all the same to you, I would say we just allocate a max size APDU of data, to allow us the most flexibility (255 bytes)

Reply:
I hear you, but this is space that will be reserved for all future versions, which may reduce the utility of stuff like phonon. Do you or @afkamalipour know how much storage there is?

Reply:
Karl has the exact numbers somewhere, but its on the order of 10KB IIRC. I don't think the difference between 255 bytes and 155 bytes will ever be noticed, but I am pretty sure trying to do an M of N multisig with only 155 bytes will be difficult

Reply:
Okay I'm cool to change it

Reply:
So for multisig, you need the addresses in the proper order, as well as the n and m values.  you simply feed these fields to OP_CHECKMULTISIG.  e.g OP_3 <address1> <address2> <address3> OP_2 OP_CHECKMULTISIG .  That being said, the size of the EEPROM is 144k and we are currently using 84k.  I don't think that allocating 255bytes will be noticed.  This will also allow some non-standard scripts.  In the longer term there is 380k ROM which we could move the program to during manufacturing freeing up all 144k for program memory.

Reply:
5486a59